corehq/apps/ota/tests/test_utils.py

```diff
@@ -1,21 +1,31 @@
 from django.test import TestCase
 
 from corehq.apps.users.models import WebUser, CommCareUser
+from corehq.apps.users.dbaccessors.all_commcare_users import delete_all_users
 from corehq.apps.users.util import format_username
 from corehq.apps.domain.models import Domain
+from corehq.apps.locations.tests.util import LocationHierarchyTestCase
 
 from corehq.apps.ota.utils import is_permitted_to_restore, get_restore_user
 
 
-class RestorePermissionsTest(TestCase):
+class RestorePermissionsTest(LocationHierarchyTestCase):
     domain = 'goats'
     other_domain = 'sheep'
+    location_type_names = ['country', 'state']
+    location_structure = [
+        ('usa', [
+            ('ma', []),
+        ]),
+        ('canada', [
+            ('montreal', []),
```

Comment:
ahem... Canada doesn't have "states". And Montreal would not be one of them even if it did.
:trollface: